dev-client/components/AppBar.tsx

```diff
@@ -19,7 +19,7 @@ export default function AppBar({title}: Props): JSX.Element {
         width="100%"
         maxW="350">
         <HStack alignItems="center">
-          <MaterialIcon
+          <MaterialIconButton
```

Comment:
Unfortunately I was a bit sloppy and renamed this component in this PR, so there's a bunch of files changed with the component name change. These can be ignored.